clis/teliod/src/qnap.rs

```diff
@@ -0,0 +1,398 @@
+use std::{
+    fs, io,
+    path::PathBuf,
+    process::{Command, Stdio},
+    time::Duration,
+};
+
+use const_format::concatcp;
+use rust_cgi::{http::Method, text_response, Request, Response};
```

Comment:
There are many places in this file where you call `text_response` with a numeric value, please convert them to https://docs.rs/http/1.0.0/http/status/struct.StatusCode.html

Reply:
Nice! changed